src/test/regress/sql/multi_alter_table_add_foreign_key.sql

```diff
@@ -0,0 +1,324 @@
+--
+-- MULTI_FOREIGN_KEY
```

Comment:
Why not add these in current multi_alter_table_add_constraints_without_name file? Is the reason just to not overpopulate that test file?

In that case, it would be good to rename this to `multi_alter_table_add_foreign_key_without_name`, or extend the test comment, to say this is just about supporting adding foreign keys without a name.